pkg/reconciler/v1alpha1/serverlessservice/serverlessservice.go

```diff
@@ -275,21 +275,21 @@ func (r *reconciler) reconcilePublicEndpoints(ctx context.Context, sks *netv1alp
 		want.Subsets = srcEps.Subsets
 
 		if !equality.Semantic.DeepEqual(want.Subsets, eps.Subsets) {
-			logger.Infof("Public K8s Endpoints changed; reconciling: %s", sn)
+			logger.Infof("Public K8s Endpoints changed; reconciling %q", sn)
 			if _, err = r.KubeClientSet.CoreV1().Endpoints(sks.Namespace).Update(want); err != nil {
-				logger.Errorw(fmt.Sprintf("Error updating public K8s Endpoints %s: ", sn), zap.Error(err))
+				logger.Errorw(fmt.Sprintf("Error updating public K8s Endpoints %q", sn), zap.Error(err))
 				return err
 			}
 		}
 	}
 	if r := presources.ReadyAddressCount(eps); r > 0 {
-		logger.Infof("Endpoints %s/%s has %d ready  endpoints", r)
+		logger.Infof("Endpoints %q has %d ready endpoints", sn, r)
```

Comment:
Hm, I don't know how this slipped, since this should not compile with modern go compiler.
But, the idea was to include namespace.